Source/WebCore/dom/Node.h

```diff
@@ -742,10 +745,10 @@ class Node : public EventTarget {
     mutable uint32_t m_refCountAndParentBit { s_refCountIncrement };
     mutable OptionSet<NodeFlag> m_nodeFlags;
 
-    ContainerNode* m_parentNode { nullptr };
-    TreeScope* m_treeScope { nullptr };
-    Node* m_previous { nullptr };
-    Node* m_next { nullptr };
+    CheckedPtr<ContainerNode> m_parentNode;
+    TreeScope* m_treeScope;
```

Comment:
Why not continue to initialize `m_treeScope` to `nullptr`?

Reply:
My bad. I had converted it to a smart pointer but had to revert to a raw pointer for now. Forgot to restore the default initializer. Will fix.